languages.toml

```diff
@@ -1324,7 +1324,7 @@ source = { git = "https://github.com/the-mikedavis/tree-sitter-git-rebase", rev
 name = "regex"
 scope = "source.regex"
 injection-regex = "regex"
-file-types = ["regex"]
+file-types = ["regex", ".Rbuildignore"]
```

Comment:
Regex is almost entirely used for injections, so `/foo/` from javascript can inject the language for example. `regex` as a file-type is just here for testing and could probably be removed. This file-type probably belongs under git-ignore instead

Reply:
Ah, I see. Unfortunately the `.Rbuildignore` syntax isn't interchangeable with the `.gitignore` syntax - it's just simply one-regex-per-line and doesn't support the `.gitignore`-style `**/*` path wildcards or the `!` negation syntax.

Even if the `regex` config wasn't initially intended to support files, it seems to work quite well for this case. If there isn't a more appropriate option, I'd be in favor of leaving it as is for now even if it is eventually removed. 

If you disagree, though, I'd be fine removing it also. Your call.

Reply:
I thought this grammar might struggle with newlines and multiple regexes but if it works well then I think it's ok 👍